server/controllers/finance/profitCenter.js

```diff
@@ -102,8 +106,43 @@ function lookupProfitCenter (id, codes) {
     });
 }
 
+function getProfitValue (req, res, next){
+
+  var sql = null, optionalCondition = '';
+
+  lookupProfitCenter(req.params.id, req.codes)
+    .then(function (){      
+      sql = 
+        'SELECT ac.id FROM account AS ac WHERE ac.pc_id=? AND ac.is_title =?';
+
+      return db.exec(sql, [req.params.id, 0]);
+    })
+    .then(function (rows){
+      if (rows.length > 0) {
+        rows = rows.map(function (row) { return row.id;});
+        optionalCondition = ' OR %table%.account_id IN (' + rows.join(',') + ')';
+      }
+
+      sql =
+        'SELECT SUM(t.credit_equiv - t.debit_equiv) as profit ' +
```

Comment:
The posting journal data is volatile - this query should only look at the general ledger.  The client-side will have to inform the user that all "unposted" data (data not in the general ledger) will not show up.